doc/classes/Array.xml

```diff
@@ -218,6 +218,11 @@
 			<param index="1" name="before" type="bool" default="true" />
 			<description>
 				Finds the index of an existing value (or the insertion index that maintains sorting order, if the value is not yet present in the array) using binary search. Optionally, a [param before] specifier can be passed. If [code]false[/code], the returned index comes after all existing entries of the value in the array.
+				[codeblock]
+				var array = [1, 2, 3, 3, 4, 5]
+				print(array.bsearch(3, true))  # Prints 2, at the first matching element.
+				print(array.bsearch(3, false)) # Prints 4, after the last matching element.
```

Comment:
Would it be better to use array elements which are not numbers to avoid the overlap between "4" being both the index and the indexed value in the array?
```suggestion
				var array = ["a", "b", "c", "c", "d", "e"]
				print(array.bsearch("c", true))  # Prints 2, at the first matching element.
				print(array.bsearch("c", false)) # Prints 4, after the last matching element.
```

Reply:
Good point! Will do